spec/features/content_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "Content Item Details", type: :feature do
```

Comment:
I think I'll leave this one as is. The content is being tested in the view test. This test is checking that you are in the right place, the right page, whether it's blank or not.